omod/src/main/java/org/openmrs/module/kenyaemr/page/controller/admin/AutoUpdatePageController.java

```diff
@@ -0,0 +1,154 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.module.kenyaemr.page.controller.admin;
+
+import org.codehaus.jackson.map.ObjectMapper;
+import org.codehaus.jackson.node.ObjectNode;
+import org.openmrs.module.kenyaemr.EmrConstants;
+import org.openmrs.module.kenyaemr.util.ServerInformation;
+import org.openmrs.module.kenyaui.annotation.AppPage;
+import org.openmrs.ui.framework.page.PageModel;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.io.IOException;
+import java.net.HttpURLConnection;
+import java.net.URLConnection;
+import java.text.DateFormat;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+
+/**
+ * Manage auto- update page for the admin app
+ */
+@AppPage(EmrConstants.APP_ADMIN)
+public class AutoUpdatePageController {
+    private static String latestReleaseUrl = "https://api.github.com/repos/palladiumkenya/kenyahmis-releases/releases/latest";
```

Comment:
I suggest this goes into a global property. Please use the config.xml to add this